frontend/app/api/hal-api-resource.js

```diff
@@ -62,6 +62,22 @@ module.exports = function HALAPIResource($timeout, $q, PathHelper) {
       return new Hyperagent.Resource({
         url: PathHelper.appBasePath + PathHelper.apiV3 + '/' + uri,
       });
+    },
+    setupLink: function(uri, params) {
+      // some monkey-patching here
+      // since the Hyperagent.Link constructor
+      // behaves differently from the links constructed
+      // from the API response
+      HALAPIResource.configure();
+      var url = PathHelper.appBasePath + PathHelper.apiV3 + '/' + uri;
```

Comment:
It is the same as above in `setup`. As the `HALAPIResource` object is of our own making this should be okay (for now).

I'll try to refactor a bit, as the `setupLink` method is imho a duplicate and `configure` should not be part of a public API to `HALAPIResource`